AcadeMeData/tests/test_client.py

```diff
@@ -1,13 +1,18 @@
 from AcadeMe import views
+from django.template.loader import render_to_string
 
 
 def test_homepage(client):
     request = client.get('')
     response = views.homePage(request)
+    expected_html = render_to_string('landing/homepage.html', request=request)
     assert response.status_code == 200
+    assert response.content.decode() == expected_html
 
 
 def test_contactus(client):
     request = client.get('contact_us/')
     response = views.contact(request)

```

Comment:
There are two checks here
response=client.get('contact_us/')
Line 15 is another test but it is not relevant here,
My mistake in the review it was approved

Reply:
Split to two test files, and commited the changes you mentioned above. @Yarboa